lib/engine/game/g_18_sj/step/buy_train.rb

```diff
@@ -19,6 +19,28 @@ def actions(entity)
           end
 
           def do_after_buy_train_action(_action, _entity); end
+
+          def buyable_trains(entity)
+            if entity.player == @game.edelsward
+              result = @depot.depot_trains[0..1]
```

Comment:
result is never used?

Reply:
This code was a remain from when I tried to get this to work.

The final way was that as part of the cleanup after dividend, a corporation owned (requisited 100%) by the bot (Edelswärd) automatically gets 0-2 trains without using buy_trains. So a bot will not use that step.